NightscoutServiceKit/RemoteCommands/Notifications/RemoteNotification.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  RemoteNotification.swift
+//  NightscoutUploadKit
+//
+//  Created by Bill Gestrich on 2/25/23.
+//  Copyright © 2023 Pete Schwamb. All rights reserved.
+//
+
+import Foundation
+
+protocol RemoteNotification: Codable {
+    
+    var id: String {get}
+    var expiration: Date? {get}
+    var sentAt: Date? {get}
+    var remoteAddress: String {get}
+    
+    func toRemoteCommand(otpManager: OTPManager, commandSource: RemoteCommandSource) -> NightscoutRemoteCommand
+    static func includedInNotification(_ notification: [String: Any]) -> Bool
+}
+
+extension RemoteNotification {
+    
```

Comment:
A unique id will be more useful with remote 2.0. For now, trying to extract something that helps with uniqueness for a remote 1.0 push notification.